lib/search/view/search_page.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:mi_libro_vecino/ui_utils/colors.dart';
+import 'package:mi_libro_vecino/ui_utils/constans/assets.dart';
+
+class SearchPage extends StatelessWidget {
+  const SearchPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: PColors.purple,
+      appBar: AppBar(
+        leadingWidth: 120,
+        leading: const Padding(
+          padding: EdgeInsets.symmetric(vertical: 22),
+          child: Image(
+            image: AssetImage(Assets.logo),
+          ),
+        ),
+        actions: [
+          TextButton(
+            onPressed: () {},
+            child: Text(
+              'Soy biblioteca',
+              style: Theme.of(context).textTheme.subtitle1!.apply(
+                    color: PColors.white,
+                    fontWeightDelta: 1,
+                  ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.symmetric(vertical: 20, horizontal: 40),
+            child: Container(
+              width: 158,
+              height: 40,
+              clipBehavior: Clip.antiAlias,
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(20),
+              ),
+              child: ElevatedButton(
+                onPressed: () {},
+                child: Text(
+                  'Siguenos',
+                  style: Theme.of(context).textTheme.subtitle1!.apply(
+                        color: PColors.white,
+                        fontWeightDelta: 1,
+                      ),
+                ),
+              ),
+            ),
+          ),
+        ],
+      ),
+      body: Center(
+        child: Container(
+          padding: const EdgeInsets.fromLTRB(20, 0, 20, 80),
+          width: 800,
+          constraints: BoxConstraints(
+            minWidth: MediaQuery.of(context).size.width * 0.5,
+          ),
+          child: SingleChildScrollView(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(
+                  'Amet minim mollit non deserunt est sit aliqua.',
+                  style: Theme.of(context).textTheme.button!.apply(
+                        color: PColors.white,
+                        fontSizeDelta: 4,
+                      ),
+                  textAlign: TextAlign.center,
+                ),
+                const SizedBox(height: 10),
+                Text(
+                  '¿Donde quieres buscar librerías?',
+                  style: Theme.of(context).textTheme.headline1!.apply(
+                        color: PColors.white,
+                        fontSizeDelta: 2,
+                      ),
+                  textAlign: TextAlign.center,
+                ),
+                const SizedBox(height: 40),
+                Text(
+                  // ignore: lines_longer_than_80_chars
```

Comment:
Instead of avoid this linter, you can divide the text in multiple lines. But it doesnt apply in this case because we are going to put texts in another file. So here you will call the variable.